backend/src/main.ts

```diff
@@ -46,10 +47,20 @@ async function bootstrap() {
     app.use(passport.initialize())
     app.use(passport.session())
     app.use((req, res, next) => {
-        console.log("Session id: ", req.sessionID)
+        console.log('Session id: ', req.sessionID)
         next()
     })
 
+    const config = new DocumentBuilder()
+        .setTitle('CosmicPong API')
+        .setDescription(
+            'The coolest way to play pong and test our skills as developers at the same time'
+        )
+        .setVersion('1.0')
+        .build()
+    const document = SwaggerModule.createDocument(app, config)
+    SwaggerModule.setup('api', app, document)
```

Comment:
We already have a route on `/api` to feed the database, so I suggest to rename the route for swagger or rename the route for feeding the database
```suggestion
    const config = new DocumentBuilder()
        .setTitle('CosmicPong API')
        .setDescription(
            'The coolest way to play pong and test our skills as developers at the same time'
        )
        .setVersion('1.0')
        .build()
    const document = SwaggerModule.createDocument(app, config)
    SwaggerModule.setup('swagger', app, document)
```